runtime/queries/help/highlights.scm

```diff
@@ -0,0 +1,15 @@
+(headline) @text.title
+(column_heading) @text.title
+(tag
+   "*" @conceal (#set! conceal "")
+   name: (_) @label)
+(option
+   "'" @conceal (#set! conceal "")
```

Comment:
```suggestion
```
Quotes around options are not concealed in the syntax file (this was just me getting carried away in nvim-treesitter...)